contracts/CompoundGovernor.sol

```diff
@@ -1,5 +1,199 @@
+        );
+    }
+
+    /// @dev returns executor address
+    /// @return address of the executor.
+    function _executor()
+        internal
+        view
+        override(GovernorTimelockControlUpgradeable, GovernorUpgradeable)
+        returns (address)
+    {
+        return address(this);
+    }
+
+    /// @dev Queues a proposal to be executed after it has succeeded.
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    /// @param _proposalId The id of the proposal.
+    /// @param _targets A list of target addresses for calls to be made in the proposal.
+    /// @param _values A list of values (ETH) to be passed to the calls in the proposal.
+    /// @param _calldatas A list of calldata for the calls in the proposal.
+    /// @param _descriptionHash The hash of the description for the proposal.
+    /// @return The id of the proposal.
+    function _queueOperations(
+        uint256 _proposalId,
+        address[] memory _targets,
+        uint256[] memory _values,
+        bytes[] memory _calldatas,
+        bytes32 _descriptionHash
+    ) internal virtual override(GovernorTimelockControlUpgradeable, GovernorUpgradeable) returns (uint48) {
+        return GovernorTimelockControlUpgradeable._queueOperations(
+            _proposalId, _targets, _values, _calldatas, _descriptionHash
+        );
+    }
+
+    /// @inheritdoc GovernorPreventLateQuorumUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function proposalDeadline(uint256 _proposalId)
+        public
+        view
+        virtual
+        override(GovernorPreventLateQuorumUpgradeable, GovernorUpgradeable)
+        returns (uint256)
+    {
+        return GovernorPreventLateQuorumUpgradeable.proposalDeadline(_proposalId);
+    }
+
+    /// @inheritdoc GovernorTimelockControlUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function proposalNeedsQueuing(uint256 _proposalId)
+        public
+        view
+        virtual
+        override(GovernorTimelockControlUpgradeable, GovernorUpgradeable)
+        returns (bool)
+    {
+        return GovernorTimelockControlUpgradeable.proposalNeedsQueuing(_proposalId);
+    }
+
+    /// @inheritdoc GovernorSettingsUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function proposalThreshold()
+        public
+        view
+        virtual
+        override(GovernorSettingsUpgradeable, GovernorUpgradeable)
+        returns (uint256)
+    {
+        return GovernorSettingsUpgradeable.proposalThreshold();
+    }
+
+    /// @inheritdoc GovernorTimelockControlUpgradeable
+    /// @dev We override this function to resolve ambiguity between inherited contracts.
+    function state(uint256 _proposalId)
+        public
+        view
+        virtual
+        override(GovernorTimelockControlUpgradeable, GovernorUpgradeable)
```

Comment:
small nit, @jferas 

usually it's the rightmost contract whose functionality is relevant / "super". I lightly prefer putting the functionality we're inheriting from as the last contract in the override args, to be a little more idiomatic. this applies to all the overridden functions.

Reply:
Done: c5184846859cc296854041bdd0feb18b2802b84a